osu.Server.Spectator/Hubs/MultiplayerHub.cs

```diff
@@ -117,6 +153,9 @@ public async Task LeaveRoom()
                         active_rooms.Remove(room.RoomID);
                     }
 
+                    using (var conn = Database.GetConnection())
+                        await conn.ExecuteAsync("UPDATE multiplayer_rooms SET ends_at = NOW() WHERE id = @RoomID", new { room.RoomID });
```

Comment:
I don't like this way of creating the object - it's too magic for my liking and relying on what I can only assume is .NET property ordering. Would you be fine if I changed these to be implemented like `new { RoomID = room.RoomID })`?

The official docs do this: https://dapper-tutorial.net/async

The difficulty calculator does this:
https://github.com/ppy/osu-difficulty-calculator/blob/0e4bc6bf8ca1536636110a8a4b10c8ac8674743b/osu.Server.DifficultyCalculator/ServerDifficultyCalculator.cs#L81-L91

Reply:
Made the changes anyway. As it is it can/will fail if the name of any of these properties change.